source/resampling_with_code.md

```diff
@@ -1186,35 +1201,37 @@ s <- 17
 s == 17
 ```
 
-::: {.callout-note}
-::: Python
+:::: {.callout-note}
+::: python
```

Comment:
The `:::` fence is treated a bit different. They need to be larger number of colons on the outside and they currently need to match the number on the closing of the fence.